test/TestSites/CreateDefaultBuilderApp/Program.cs

```diff
@@ -15,24 +14,28 @@ public class Program
     {
         static void Main(string[] args)
         {
-            string responseMessage = string.Empty;
+            string responseMessage = null;
 
             WebHost.CreateDefaultBuilder(new[] { "--cliKey", "cliValue" })
-                .ConfigureServices((context, services) =>
+                .ConfigureServices((context, services) => responseMessage = responseMessage ?? GetResponseMessage(context))
+                .ConfigureKestrel(options =>
                 {
-                    responseMessage = GetResponseMessage(context, services);
-                })
-                .Configure(app =>
-                {
-                    app.Run(context =>
+                    if (responseMessage == null
```

Comment:
change this to use the [Endpoint](https://github.com/aspnet/KestrelHttpServer/blob/e958d8258445c2cda73c244d586f06ca6753be4e/samples/SampleApp/Startup.cs#L147-L151) API to make sure the endpoints actually get loaded:
options.Endpoint("HTTP", opt =>
                        {
                            opt.ConfigSection["KestrelEndPointSettingName"] ...
                        })